typescript-json-schema.ts

```diff
@@ -372,7 +380,7 @@ export class JsonSchemaGenerator {
             }
         };
 
-        enm.members.forEach(member => {
+        members.forEach(member => {
```

Comment:
See how I implemented it here: https://github.com/vega/typescript-to-json-schema/commit/462fe6d6a9ef7a52cd56edb8cfe930dd9f466a54. 

Btw, thanks for the PR. It helped me to also fix a bug in the other library I'm working on.